src/main/java/fr/ospiea/oscra/employee/object/Role.java

```diff
@@ -0,0 +1,10 @@
+package fr.ospiea.oscra.employee.object;
+
+/**
+ * Created by taozheng on 19/09/2016.
+ */
+public enum Role {
```

Comment:
That's why I named it 'user'
a user can have a role like 'employee'. or a role 'manager'. 
I don't understand why you rename 'user' by 'employee'